src/m_config.c

```diff
@@ -1254,6 +1260,13 @@ static default_t extra_defaults_list[] =
 
     CONFIG_VARIABLE_INT(joystick_x_invert),
 
+    //!
+    // Joystick x axis analog sensitivity, specified as a value between 0 and
+    // 20.
+    //
+
+    CONFIG_VARIABLE_INT(joystick_x_sensitivity),
```

Comment:
Why does this exist? There is no corresponding y sensitivity and there are separate move and look sensitivities below.

Reply:
Once we merge this the config file format is pretty fixed so we want to make sure it's perfect before we do.

I think we want to have explicit and separate sensitivities for turning and looking. Do other have opinions on this?

Reply:
Indeed, we have separate sensitivities for moving, turning and looking by controller in Woof!.

Reply:
Okay, the variable is now `joystick_turn_sensitivity`.

Reply:
Now that I've made it so 0 sensitivity disables inputs in non-analog mode, I've restored the look sensitivity option in chocolate-strife-setup.